content/docs/app-developer-guide/export-to-oci-layout.md

```diff
@@ -0,0 +1,151 @@
+If you are interested on keep playing with the image in  OCI layout format, one tool you can take a look is [umoci](https://umo.ci/) it can help you to create a 
+[runtime bundler](https://github.com/opencontainers/runtime-spec) that can be executed with another tool like [runc](https://github.com/opencontainers/runc)
+
+---
+
+## Extra configuration
+
+### Skip saving your run-image layers on disk
+
+Before using this option we suggest to remove your local layout directory (the one configured in your pack config.toml with the key `layout-repo-dir`) and 
+your application image folder (if you are planning to use the same folder), the reason for this is pack doesn't remove the blobs saved in the `layout-repo-dir` if you use the `--sparse` flag 
+
+If you don't need your `run-image` layers on disk, you can skip them using `--sparse` flag in your `pack build` command invocation.
+
+For example:
+
+```bash
+pack build oci:sample-app --sparse --path samples/apps/java-maven --builder cnbs/sample-builder:bionic
+```
+
+Verify your application image
+
+```bash
+sample-app
+├── blobs
+│ └── sha256
+│     ├── 2ebed3ab57806441e2bf814eaf0648ed77289e058340d2b76d32b422fbaac5d8
+│     ├── 2fa192256ce255c6ea6c1296eadfe2feba8094f40e6aa85e699645caca2e85d8
+│     ├── 5a44e4f7b58d74fe6f92dd7028075c91191128d1e2e7f39846fe061a9a98836e
+│     ├── 741e558b7b807fea350b26b8152170a2463277cb3d1268b60de76ec12608518a
+│     ├── 907c84671180d979a38affb62d9a6ea8e9a510e27639e0b60a34a42f1a846ddc
+│     ├── ad13830c92258c952f25d561d8bf7d9eb58b8a3003960db1502cbda8239130b5
+│     ├── c44222730efa142cd5bedc0babf82a9a07d325494be7f5c3cfde56f43166b65f
+│     └── f9d6350d0c44c0e7165a522155f53181ce8c163a6b8ead1f6baea22d1a8d8a78
+├── index.json
+└── oci-layout
+
+3 directories, 10 files
+```
+
+As you can see, there are 3 missing files at `sample-app/blobs/sha256` folder, the missing 3 blobs are the blobs from the 
+`run-image` that were not downloaded but if you check your config file you'll notice you have the same number of layers as 
+when you export the full image.
+
+## Implementation notes
+
+### Media Types
+
+According to the OCI specification, the [compatibles media types](https://github.com/opencontainers/image-spec/blob/main/media-types.md#compatibility-matrix) for the index.json files must be:
+
+- `application/vnd.oci.image.index.v1+json` or
+- `application/vnd.docker.distribution.manifest.list.v2+json` 
+
+If you are trying to use the lifecycle directly without using `pack` to export your image, take on consideration that tools like:
+
+[skopeo](https://github.com/containers/skopeo)
+```bash
+skopeo copy -a docker://<your-image> <dest>
+```
+It will give you `application/vnd.oci.image.index.v1+json` media type, which is currently working
+
+But [crane](https://github.com/google/go-containerregistry/tree/main/cmd/crane) 
+
+```bash
+crane pull <your-image> <dest> --format=oci
+```
+It will give you `application/vnd.docker.distribution.manifest.list.v2+json`, which will fail
```

Comment:
Do these things pass/fail due to the state of _our_ current implementation?  Or because of a limitation in `crane`, or because the operation violates part of the OCI spec? Or for some other reason?

Reply:
agree, helpful to add a little more detail here

Reply:
It fails because of the state or our current implementation, it should be improve on new versions of the feature. I will add it

Reply:
Let me explain this part a little better so you can guide me on how to document it. 

According to the [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0119-export-to-oci.md#how-it-works) if you want to export your application image on disk, you need to mount to the lifecycle a *local storage* with the images require by the build process, for example, the `run-image`. This storage must contain images in OCI layout format and name resolution will follow some rules defined also in the RFC. 

The documented part was based on a feedback provided by @natalieparellano [here](https://github.com/buildpacks/rfcs/pull/203#discussion_r1092449172), which is basically the following scenario: Suppose you want to execute the lifecycle directly without using pack, for example, a platform implementor. They need to create this local storage and save everything there before executing the build, right?, well the problem is: *If they save in the local storage a manifest list* and then they execute the lifecycle, right now, we are not smart enough to detect which manifest we need to pick to execute the build. 

That is the actual comment we want to document so users can take it into consideration, now I am not sure this is the right place to do it, or maybe I didn't write it correctly, so any advice will be appreciate!